device.go

```diff
@@ -314,6 +314,7 @@ func vfioDeviceHandler(ctx context.Context, device pb.Device, spec *pb.Spec, s *
 	fieldLogger := agentLog.WithField("host-dev", device.ContainerPath)
 
 	var rebindToVfio bool
+	var group string
```

Comment:
Apparently, github has no good way to comment on the commit message.

>other, they most appear as the same IOMMU group in the guest as well.

Typo: s/most/must/.